test/lib/ex_debug_toolbar/plug/request_id_test.exs

```diff
@@ -0,0 +1,24 @@
+defmodule ExDebugToolbar.Plug.RequestIdTest do
+  use ExUnit.Case, async: true
+  use Plug.Test
+  import Plug.Conn
+
+  test "it sets request id in response headers" do
+    [request_id] = build_conn() |> get_resp_header("x-request-id")
+    assert request_id
+  end
+
+  test "it sets request id in process metadata" do
```

Comment:
this one is get_resp the next one is get_req
took me a while to notice it in request id plug